### ISSUE TITLE
[UNBLOCK] pusher.com

### ISSUE THREAD

**Issue by latelatelate** [celenityy/bugs#29]
### Which Content Blocker are you using?

uBlock Origin

### With which list(s) does the problem occur?

BadBlock

### Which domain(s) should be unblocked?

||pusher.com^$important # Pusher/Bird Marketing https://bird.com/

### Why should the domain(s) be unblocked?

Removing the ^$important tag to allow users to create their own exception to this rule might be a bit more user friendly. Blocking the root domain is also blocking legitimate apps that utilize pusher (chat on live streaming apps, etc)

lmk what you think!

**Comment by celenityy** [comment]
Thank you for the issue!

My philosophy behind adding `$important` to domains is generally for ones that are unjustifiably whitelisted by other widely used lists (like AdGuard's, ex. `google-analytics.com`). 

However, for domains that do also provide legitimate uses, those make sense to leave alone and not append `$important` to. It sounds like this is one of those cases, so I'll probably go ahead and remove `$important` from `pusher.com` (And will also make sure it's not being blocked on a DNS level by ex. AdGuard Home, we can make sure only covered in browsers).

In the mean-time, would you mind sharing examples of this domain's legitimate use case? I'll take a look & investigate into it.

**Comment by celenityy** [comment]
Migrated to our new issue tracker at https://codeberg.org/celenity/bugs/issues/29, hope to see you there!
